api-reference/50 Common/Object Structures/dxSchedulerAppointment/endDate.md

```diff
@@ -7,4 +7,7 @@ type: Date
 Specifies the ending of the appointment.
 
 ---
-The **startDate** and **endDate** fields are used to position the rectangle representing an appointment in the scheduler.
\ No newline at end of file
+We recommend that you specify appointment dates in <a href="https://en.wikipedia.org/wiki/ISO_8601" target="_blank">UTC ISO 8601</a>.
```

Comment:
Do you mean in <a href="https://en.wikipedia.org/wiki/ISO_8601" target="_blank">UTC ISO 8601</a> format?

Reply:
Yes.